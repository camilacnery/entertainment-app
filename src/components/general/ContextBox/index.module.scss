@use '@/styles/constants' as *;

.content {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  min-height: 40vh;
  border-radius: 5px;
  box-shadow: $shadow;
  background-color: $background-contrast-color;
  padding: 2rem;
  z-index: 2;
  margin: auto;

  @media screen and (max-width: 1031px) {
    width: 85%;
  }

  @media screen and (min-width: 1032px) {
    width: 40%;
  }
}

