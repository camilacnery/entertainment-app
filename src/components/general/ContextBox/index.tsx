import { FC, ReactNode } from "react";
import styles from "./index.module.scss";

type TContentBox = {
  children: ReactNode;
};

const ContentBox: FC<TContentBox> = ({ children }) => {
  return <section className={styles.content}>{children}</section>;
};

export default ContentBox;
