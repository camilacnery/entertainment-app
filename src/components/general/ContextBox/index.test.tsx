import { render } from "@testing-library/react";

import ContextBox from ".";

describe("ContextBox", () => {
  it("renders component", () => {
    const { baseElement } = render(
      <ContextBox>
        <div>Test</div>
      </ContextBox>
    );

    expect(baseElement).toMatchSnapshot();
  });
});
