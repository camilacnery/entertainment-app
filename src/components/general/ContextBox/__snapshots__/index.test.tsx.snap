// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContextBox renders component 1`] = `
<body>
  <div>
    <section
      class="content"
    >
      <div>
        Test
      </div>
    </section>
  </div>
</body>
`;
